beacon-light-client/solidity/hardhat.config.ts

```diff
@@ -53,6 +60,7 @@ const conf = {
   GNOSIS_RPC: process.env.GNOSIS_RPC || '',
   EVMOS_RPC: process.env.EVMOS_RPC || '',
   AURORA_RPC: process.env.AURORA_RPC || '',
+  LUKSO_RPC: process.env.LUKSO_RPC || '',
```

Comment:
We are okey with those values being null, as you can add only the RPCs for the networks you will use.  And because of that we dont want to assert not null - we cant use `getEnvString`